docs/tex/TestingProcedure.tex

```diff
@@ -7,24 +7,54 @@ \subsection{Acceptability}
 % Are people willing to eat output? Incl. appearance, aroma, flavor, and texture
 % Specific preparation and/or incorporation of output (formulation) for tasting
 
-% Suggestion: Double-blind volunteer study?
+% Suggestion: Double-blind volunteer study
+Tested via blind studies where participants are divided into two groups and given either control outputs (i.e. established commercial product) or test outputs (i.e. produced by PeaPod). Participants will rate outputs on 4 criteria (appearance, aroma, flavour, and texture) on a 9-point scale.
+
+Blind studies are eminent in consumer testing as they allow researchers to get a completely unbiased dataset. Special care needs to be taken when presenting, preparing, and collecting samples for testing to ensure researchers do not influence results. Ideally, resources will permit a double-blind study where researchers hire an outside entity to conduct the test and return results with generic labels.
+
+The 4 criteria and 9-point metric scale are given by !CITE which states they are !INFO
 
 \subsection{Safety of Process}
 % SPECIFICALLY food production area
 % Chemical hazards - Toxins, heavy metals (As, Cd, Hg, Pb, etc.)
 % Biohazards - total aerobic plate count, ATP testing of food contact surfaces
 % TODO: What do those^ mean? How do we test them?
+\subsubsection{Chemical Hazards}
+As per the parts manifest !LINK, all components involved in the unit are clear of toxins and heavy metals by composition.
+
+\subsubsection{Biological Hazards}
+Aerobic Plate Count (APC) testing to be done with the Conventional Plate Count Method outlined in the FDA's \href{https://www.fda.gov/food/laboratory-methods-food/bam-chapter-3-aerobic-plate-count}{BAM Chapter 3: Aerobic Plate Count}. This is selected over the Spiral Plate Method as it is inexpensive and uses many household materials. The goal of APC testing is to indicate the bacterial population in food-adjacent sections of the design. Results to be compared against STD-3001 to ensure a maximum of 3000 colony forming units/square ft. Plate count to be minimized by following !CITE (surface cleaning standards? hard to find). 
+
+ATP testing to be done using !CITE (lots of stuff about methods but no standards? look at requirements more)
 
 \subsection{Safety of Outputs}
 % SPECIFICALLY food outputs
 % Biohazards - total aerobic count, specific pathogens (enterobacteriaceae, salmonella, yeasts, molds, E. coli, Listeria, etc.)
+APC testing conducted on samples as outlined above to ensure bacterial population below 20 000 CFU/g per STD-3001.
+
+Critical pathogens to be tested for individually:
```

Comment:
E. coli and listeria should be 0 CFU/g (our tech does not need it for production, ergo we should eliminate it)

Reply:
Side note: research UV LED spectrum and intensity vs disinfection rate and/or duration-to-disinfect? does this change for different pathogens? is there a "catch-all"?